### DISCUSSION TITLE
[crazyswarm_server-2] process has died

### DISCUSSION THREAD

**Discussion by TheLastPixie** [bitcraze/discussions#410] [Q&A]
Hi everyone, 
I'm wondering if I can get some help with this issue I have seen this brought up twice before but the scenarios were different.

-  in our case we just got crazyflie 2.1 this week and the it works fine from the client.

- The problem is ROS related, we followed the installation steps exactly as shown crazyswarm repository, I have installed, and uninstalled ROS and crazyswarm multiple times but to no avail. (made sure to uninstall properly everytime)
- the crazyradio connects normally via the `rosrun crazyflie_tools scan -v`
- also one more thing the chooser.py always gives me an error when we run a battery test. dunno if that is going to help
- when we edit the yaml file to disable motion capture we get the following error:
`[crazyswarm_server-2] process has died [pid 76390, exit code -6, cmd /home/user/crazyswarm/ros_ws/devel/lib/crazyswarm/crazyswarm_server __name:=crazyswarm_server __log:=/home/user/.ros/log/0b313522-6603-11ed-b787-49934f717d49/crazyswarm_server-2.log].
log file: /home/user/.ros/log/0b313522-6603-11ed-b787-49934f717d49/crazyswarm_server-2*.log`
Ubuntu version: 20.04
ROS: Noetic

Any kind of help would be appreciated

**Comment by krichardsson** [comment]
Hi! this issue seems to be related to Crazyswarm that we don't maintain ourselves. You would be better off putting this question on Crazyswarm's discussion board if you haven't already: https://github.com/USC-ACTLab/crazyswarm/discussions

Let us know if they are able to help you with this.

**Reply by TheLastPixie** [reply]
Thanks a lot I will try there. also, any hint where might be the fault would be appreciated.

**Reply by krichardsson** [reply]
Sorry, I don't have any experience with CrazySwarm

**Reply by TheLastPixie** [reply]
@krichardsson thanks alot and no problems at all, random question, can I just connect the crazyflie to ROS without relying on Crazyswarm?

**Reply by knmcguire** [reply]
There is this old ROS package that works on Melodic: http://wiki.ros.org/crazyflie but it is very old 

Here is a page of the external projects you can use to communicate to the crazyflie: https://www.bitcraze.io/support/external-projects/
